warp/yul/FunctionPruner.py

```diff
@@ -45,4 +49,7 @@ def _is_unused_function(self, node):
         return (
             isinstance(node, ast.FunctionDefinition)
             and node.name not in self.visited_functions
+            and "constructor" not in node.name
+            and "setter" not in node.name
+            and "getter" not in node.name
```

Comment:
These are too general in that a user may write a function with these terms in it. However it's not that big a problem because this is an underaproximation of unused functions.